Samples/CameraSettingsExternalSettingsApp/ExternalCameraSettingsAppSample/ExternalCameraSettingsAppSample.csproj

```diff
@@ -0,0 +1,191 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
perhaps it is about time we stop x86 and arm32 configurations for apps.

Reply:
point taken, but these gets autogenerated so no use of manually removing them.